### DISCUSSION TITLE
CORTX Slack community joining link is no longer active

### DISCUSSION THREAD

**Discussion by rannnayy** [Seagate/cortx#1601] [Q&A] [answered]
I wanted to join the CORTX Slack community, but when the link is clicked, it says that the link is no longer active (I attached the screenshot below). I've tried clicking all links provided in the Github repo and the website, all of them gave the same results. Can someone update the link?
Thanks!

<img width="461" alt="image" src="https://user-images.githubusercontent.com/72875641/178907296-de13529e-c189-41cf-85d4-f6187b9e0240.png">

**Comment by hessio** [comment, marked as answer]
Thanks for letting us know @rannnayy I will update this immediately! 👍
